mythtv/programs/mythfrontend/manualschedule.h

```diff
@@ -60,6 +60,7 @@ class ManualSchedule : public MythScreenType
     QString m_startString;
     QString m_chanidString;
 
+    static int last_manual_chan; /* Channel used in the last manual schedule */
```

Comment:
https://www.mythtv.org/wiki/Coding_Standards#Class_and_variable_names This name should be `s_lastManualChan`, i.e. prefixed with `s_` and **lowerCamelCase**; however, I like the name `s_lastChannel` better since the manual is already implied by `ManualSchedule::`.  Or `s_lastChannelIndex` since it is an index.

(Feel free to ignore this one.) Personally, I find the separate declaration and definition syntax a little strange since you have to re-declare the type (I understand why, because the declared type could be incomplete).  Since we are using C++17, you can declare the static variable `inline` in the class with the definition.